package.json

```diff
@@ -49,29 +63,25 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-helmet": "^6.1.0",
-    "react-i18next": "^13.0.0",
-    "react-router": "5.2.0",
-    "react-router-dom": "5.2.0",
+    "react-i18next": "^11.7.3",
+    "react-router": "5.3.x",
+    "react-router-dom": "5.3.x",
```

Comment:
Do you plan to add the shared module of Router v6? `react-router-dom-v5-compat`

Reply:
Yes in the next pr